src/main/webapp/WEB-INF/jpa-configuration.xml

```diff
@@ -50,11 +50,21 @@
         </property>
     </bean>
 
-    <bean id="dataSource" class="org.springframework.jdbc.datasource.DriverManagerDataSource">
-        <property name="driverClassName" value="${org.osiam.resource-server.db.driver}"/>
-        <property name="url" value="${org.osiam.resource-server.db.url}"/>
-        <property name="username" value="${org.osiam.resource-server.db.username}"/>
-        <property name="password" value="${org.osiam.resource-server.db.password}"/>
+    <bean id="hikariConfig" class="com.zaxxer.hikari.HikariConfig">
+        <property name="poolName" value="springHikariCP" />
+        <property name="jdbcUrl" value="${org.osiam.resource-server.db.url}" />
+        <property name="driverClassName" value="${org.osiam.resource-server.db.driver}" />
+        <property name="dataSourceProperties">
+            <props>
+                <prop key="url">${org.osiam.resource-server.db.url}</prop>
```

Comment:
I would prefer this. I would also prefer to not answer this question as a comment to a totally unrelated line comment ;)